components/match2/wikis/rocketleague/match_group_input_custom.lua

```diff
@@ -207,6 +212,45 @@ function matchFunctions.getExtraData(match)
 	return match
 end
 
+function p._getCasterInformation(name, flag, id)
+	if String.isEmpty(flag) then
+		flag = Variables.varDefault(name .. '_flag')
+	end
+	if String.isEmpty(id) then
+		id = Variables.varDefault(name .. '_dn')
+	end
+	if String.isEmpty(flag) or String.isEmpty(id) then
+		local parent = Variables.varDefault(
+			'tournament_parent',
+			mw.title.getCurrentTitle().text
+		)
```

Comment:
Any correctly setup page will have `tournament_parent` defined.

Reply:
are all pages already purged?
and are we sure there are no wrongly set up pages that will break due to not having a fallback?

Reply:
It was necessary for the example I was testing on my user page

Reply:
It should only be needed if the page has neither League Infobox and HiddenDataBox

Reply:
which in user space might happen here or there
the query i mentioned above only checked on main space (as user space imo is only secondary)